database/postgresql/schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE profile_service;
```

Comment:
cant create database with postgres using node you have to create it manual in shell.  You can create and drop tables tho

Reply:
Yes, this is why I ran these commands on the PostgreSQL and Cassandra shells line by line.